phones/models.py

```diff
@@ -204,6 +204,23 @@ def relaynumber_post_save(sender, instance, created, **kwargs):
         )
 
 
+def last_inbound_date_default():
+    return datetime.now(timezone.utc)
+
+
+class InboundContact(models.Model):
+    relay_number = models.ForeignKey(RelayNumber, on_delete=models.CASCADE)
+    inbound_number = models.CharField(max_length=15)
+    last_inbound_date = models.DateTimeField(
+        default=last_inbound_date_default
```

Comment:
Do we want to use [auto_now_add](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.DateField.auto_now_add) instead? I think it does the same thing, except you may need some extra code in tests if you want to initialize to a different value than now.

Reply:
I got scared off `auto_now_add` by this note in the docs:

> The auto_now and auto_now_add options will always use the date in the [default timezone](https://docs.djangoproject.com/en/3.2/topics/i18n/timezones/#default-current-time-zone) at the moment of creation **or update**. If you need something different, you may want to consider using your own callable default or overriding save() instead of using auto_now or auto_now_add; or using a DateTimeField instead of a DateField and deciding how to handle the conversion from datetime to date at display time.

Which I took to mean anytime we update the `blocked` field of the contact, it would ALSO update the `last_inbound_date`?

Reply:
The note is poorly worded. `auto_now_add` works at creation, and `auto_add` at update (. I tried `auto_now_add` and what you suggested - sending a `PATCH` with `blocked` does _not_ update `last_inbound_date`.

Now that I think of it, shouldn't we have a bunch of `created` and `modified` fields for all these models?